src/source/view/controls/ValidationUtils.bs

```diff
@@ -0,0 +1,49 @@
+import "pkg:/source/core/BaseClass.bs"
+
+class InputValidator extends mc.BaseClass
+    '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    '++ Private Fields
+    '+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    private NUMERIC_VALIDATOR_TYPE = "numeric"
+    private EMAIL_VALIDATOR_TYPE =  "email"
```

Comment:
never SNAKE_CASE - always camelCase please.

for enums, we have proper enum support now. We need to move this to bsc 1, so we can use that. For now, let's use _fake enums.

so make a class in this file

class ValidationTypes
  numeric = "numeric"
  email = "email"
end class


in your InputValidator class, instantiate this fake enum

private validationTypes = new mv.controls.ValidationTypes()

then below..
if validationType = m.validationTypes.numeric
...
end if


I like this pattern as it's clea rin the code, and will be easy to change to an enum when we move to bsc 1 (which is soon).